eox_tenant/templatetags/ednx.py

```diff
@@ -154,3 +158,49 @@ def get_login_link():
         return settings.FEATURES.get('ednx_custom_login_link')
     else:
         return get_lms_root_url() + "/login"
+
+
+@register.simple_tag(name="tenant_css_overrides_file")
+def tenant_css_overrides_file():
+    """
+    Django template tag that outputs the css import for a:
+    {% tenant_css_overrides_file %}
+    """
+    if get_language_bidi():
+        file_path = configuration_helpers.get_value(
+            'css_overrides_file_rtl',
+            configuration_helpers.get_value('css_overrides_file')
+        )
+    else:
+        file_path = configuration_helpers.get_value('css_overrides_file')
+
+    if file_path is not None:
+        return "<link href='{}' rel='stylesheet' type='text/css'>".format(static(file_path))
+    else:
+        return ""
+
+
+@register.simple_tag(name="tenant_rtl")
+def tenant_rtl_tag():
+    """
+    Django template tag that outputs the direction string for rtl support
+    """
+    return 'rtl' if get_language_bidi() else 'ltr'
+
+
+@register.filter
+def tenant_template_path(template_name):
```

Comment:
I am not sure if we need this function

Reply:
@morenol now I'm using the get_template_path method from theming helpers which is a proxy to hide microsite.get_template_path behind comprehensive theming